front/src/applications/stdcmV2/components/StdcmResults/StdcmResultsTable.tsx

```diff
@@ -120,7 +112,7 @@ const StcdmResultsTable = ({
           })}
         </tbody>
       </table>
-      <div className="display-all" style={!isSimulationSelected ? {} : { borderRadius: '0' }}>
+      <div className="display-all" style={!isSimulationRetained ? {} : { borderRadius: '0' }}>
```

Comment:
We should rename this class with something like that 

```suggestion
      <div className="results-buttons" style={!isSimulationRetained ? {} : { borderRadius: '0' }}>
```

Reply:
Done